libnopegl/src/backends/vk/pipeline_vk.c

```diff
@@ -602,6 +609,9 @@ static VkResult create_pipeline(struct pipeline *s)
     } else {
         ngli_assert(0);
     }
+
+    s_priv->pipeline_bind_point = bind_point_map[s->type];
```

Comment:
do you plan to unify the codepaths calling `vkCmdBindPipeline` or is there another rationale here?

Reply:
I plan to unify the codepaths later on.

Reply:
Can you mention that in the commit description?